operations/kubernetes/Repopulate_Data_in_etcd_Clusters_When_Rebuilding_Them.md

```diff
@@ -37,7 +37,7 @@ Reconstruct boot session templates for impacted product streams to repopulate da
 Boot preparation information for other product streams can be found in the following locations:
 
 - UANs: Refer to the UAN product stream repository and search for the "PREPARE UAN BOOT SESSION TEMPLATES" header in the "Install and Configure UANs" procedure.
```

Comment:
Another instance where UAN is referred to as a separate product stream.  Might be a good idea to grep the entire repo for this.